packages/wdio-config/src/node/ConfigParser.ts

```diff
@@ -185,6 +187,13 @@ export default class ConfigParser {
             this._config.exclude = object.exclude as string[]
         }
 
+        /**
+         * add this check to avoid duplication of the "services" in the config
+         */
+        if (!addServicesToConfig) {
+            this._config.services = services
```

Comment:
Instead of doing that I suggest to use [`customMerge`](https://www.npmjs.com/package/deepmerge#custommerge) and skip merging for `services` and `reporters`. I think we can generally avoid to merge all arrays here.

Reply:
Do you mean we have to remove the flag and implement customMerge?

Reply:
Yeah, I would suggest using customMerge option as it makes it easier. Note that we also need this for `reporters` as it is the same use case